include/glow/Graph/Graph.h

```diff
@@ -261,10 +270,78 @@ class Function final : public Named {
 
   const Module *getParent() const { return parent_; }
 
+  /// Logs the node creation with a list of input nodes.
+  void logNodeCreation(Node *newNode, llvm::ArrayRef<Node *> inputs) {
```

Comment:
You could move these functions into the logContext to avoid adding all this log specific code into graph.h

Reply:
moved to Log.cpp now